ovos_tts_plugin_piper/util.py

```
@@ -0,0 +1,713 @@
+def pronounce_time(time_string: str, full_lang: str) -> str:
+    """
+    Pronounces a time string using ovos-date-parser.
+    Handles military time like "15h01" and converts it to a
+    datetime.time object before passing it to nice_time.
+    """
+    try:
+        hours, mins = time_string.split("h")
+        time_obj = datetime.time(int(hours), int(mins))
+        # Use nice_time from ovos-date-parser
+        return nice_time(time_obj, full_lang, speech=True, use_24hour=True, use_ampm=False)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)